### PR TITLE
feat(FX-3514): remove exhibitors logo from list on Exhibitors A-Z tab

### PR DESCRIPTION
**Pull request by tr-ann**
Jira: [FX-3514](https://artsyproduct.atlassian.net/browse/FX-3514)

### Description
Remove the logo thumbnails from the AZ tab on the Fairs page.
Examples: https://www.artsy.net/fair/2021-art-taipei/exhibitors

Why? Too many exhibitors do not have a profile logo uploaded. When they do, it is too often getting cropped in a way that makes it illegible.
As a result, this page is looking buggy and unprofessional. We have already received complaints from fair organizers about it. Since many people view this page it also affects our brand.

It seems the easiest way around this would be to remove the thumbnails completely, just leave the gallery name and follow button.

### Changes
- updated graphql query
- removed logo component

### Demo

https://user-images.githubusercontent.com/56556580/141275756-94a957ee-2bbb-48f9-af18-8b7c96aec91f.mov

### CODE REVIEW

**Comment by mdole** [comment]
Let's confirm that Product/Design are good with this solution before merging! Asked to confirm in the Jira ticket: https://artsyproduct.atlassian.net/browse/FX-3514?focusedCommentId=43503

**Review by dimatretyak — APPROVED** [review]
LGTM. Let's wait for a response from the designers

**Comment by tr-ann** [comment]
@mdole Lois approved this changes in the Jira ticket. Can we merge it when all checks will be 🟢 ?

**Comment by mdole** [comment]
awesome, yeah! Merging!

**Pull request merged**
